doc/nrf/protocols/matter/getting_started/manufacturer_cluster_editor_tool.rst

```diff
@@ -7,6 +7,8 @@ nRF Connect Matter Manufacturer Cluster Editor tool
    :local:
    :depth: 2
 
+.. include:: /includes/matter_manufacturer_cluster_editor_note.txt
+
```

Comment:
Line 31: Remove the word command after the link (link resolves to zap-append command)